src/main/java/in/org/projecteka/hiu/SecurityConfiguration.java

```diff
@@ -34,7 +36,6 @@
 
     private static final List<Map.Entry<String, HttpMethod>> SERVICE_ONLY_URLS = new ArrayList<>() {
         {
-            add(Map.entry("/consent/notification", HttpMethod.POST));
```

Comment:
Is there any reason to remove /consent/notification API from SERVICE_ONLY_URLS list?

Reply:
added again